### PR TITLE
set download links to point always towards the latest release

### PR DESCRIPTION
**Pull request by Lisiadito**
This change must NOT be merged before the naming convention of the release is changed. 
See discussion in #8

### CODE REVIEW

**Comment by Jocs** [comment]
@Lisiadito Thanks. you also need to `npm run build` to build the package.

**Comment by Lisiadito** [comment]
Ups sorry forgot that. Will do that.

**Comment by Jocs** [comment]
@Lisiadito My fault, I did not mention it in the README file.

**Comment by fxha** [comment]
@Lisiadito Version 0.16.1 has been release, please rebase the PR if you have time.

File: src/components/titleBar.vue [hunk 1]
```diff
@@ -24,19 +24,19 @@
         <div class="download-group">
           <div class="download-info">Available for macOS, Windows and Linux.</div>
           <div class="button-group">
-            <a class="button" href="https://github.com/marktext/marktext/releases/download/v0.15.1/marktext-0.15.1.dmg" target="_blank">
+            <a class="button" href="https://github.com/marktext/marktext/releases/download/latest/marktext.dmg" target="_blank">
```
Comment (fxha) [inline src/components/titleBar.vue:27]: @Lisiadito According to [this](https://help.github.com/en/github/administering-a-repository/linking-to-releases), the correct URL is `https://github.com/marktext/marktext/releases/latest/download/[...]`

Comment (Lisiadito) [reply src/components/titleBar.vue:27]: true... will fix it.

**Pull request merged**